app/test_main.py

```diff
@@ -1 +1,64 @@
-# write your code here
+from typing import Callable
+from unittest import mock
+import pytest
+from app.main import can_access_google_page
+
+
+@pytest.fixture()
+def mock_valid_google_url() -> None:
```

Comment:
invalid return typehint
https://stackoverflow.com/questions/43658999/what-is-the-return-type-hint-of-a-generator-function